Dockerfile

```diff
@@ -1,17 +1,36 @@
-FROM node:18.6
+FROM node:18.18
 
-# Install chrome and necessary dependencies
+# Install necessary dependencies
 RUN apt-get update \
-    && apt-get install -y wget gnupg ca-certificates jq \
-    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && apt-get install -y wget gnupg ca-certificates jq expect
+
+# Install Chrome
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list' \
     && apt-get update \
     && apt-get install -y google-chrome-stable
 
+# Setup Golang
+COPY --from=golang:1.21 /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV GOPATH="/go"
+ENV GOBIN="/go/bin"
+
+# Agoric SDK Setup
```

Comment:
why is this building an SDK image? I'd expect it to use https://github.com/Agoric/agoric-sdk/pkgs/container/agoric-sdk ?

Reply:
I was not aware of the SDK image. After using the SDK image, the CI time is reduced to half. 

But for some reason, the tests for placing bids are failing. I am inspecting the issue. 

Apparently, it says:
```
Error: key with address randomWalletAddress... not found: key not found
```
The wallet address exists and all the tests work fine with the old setup. I'm missing some steps?

Reply:
I wasn't aware of it. Using the SDK image has improved the CI time. It's been halved now. Thanks for sharing!

But bidding-specific test cases are failing now for some reason. I am inspecting the issue and will get back to you once done with finding the root cause.

Reply:
@turadg, I was investigating the issue and discovered that it's related to a discrepancy in node versions. 
The `agoric-sdk` image is configured with node `18.20`and presently, our tests run smoothly with node version `18.18`. I was able to replicate the failing test cases in our CI environment by upgrading my local node version to 18.20.

It's more so of an issue with `cypress` because inside the container I was successfully able to place bids from the shell. 
I was trying to update `cypress` version for `@agoric/synpress` and make it compatible with node `18.20`. But I feel it needs to be done carefully to avoid breaking other things. 

Do you have any suggestions? Can we override the node version in `agoric-sdk` image?

Reply:
@turadg Finally working. I've used `nvm` to install node `18.18`. When I run tests, I switch to node `18.18` with `nvm use` and it works.

Reply:
ok, if the tests require 18.18 to run please,
- add a `.node-version` file that some Node version managers will observe automatically (I use `nodenv` which does)
- file an issue on the repo to remove that restriction (it may just require updating `ses` and @endo` deps to latest)

Reply:
@turadg I think `ses` version is not an issue. I tried different versions and it gave no issues until I switched to node 18.20.

I also tried bumping versions of `cypress` and `typescript`. But they work fine till I use node 18.20. The issue doesn't exist for node versions up to 18.18. For versions beyond 18.18, excluding 20.4, the majority exhibit this issue.

The issue is the following error when trying to place bids with the `agops` command:

```
Removing intrinsics.Symbol.dispose
failed to delete intrinsics.Symbol.dispose (TypeError#1)
TypeError#1: Cannot delete property 'dispose' of function Symbol() { [native code] }

(TypeError#1)
```

I came across it over here as well:
https://github.com/Agoric/agoric-sdk/issues/8599